pegasus/sites.v3/code.org/public/ops-dashboard/cohort-edit.haml

```diff
@@ -75,3 +58,13 @@ theme: none
   .form-group.action-buttons
     %button#btn-save.btn.btn-default{type: "submit", "ng-click" => "save(cohort)"} Save
     %button#btn-cancel.btn.btn-link{"ng-click" => "cancel_edit()"} Cancel
+
+:css
+  .row {
+    padding: 10px;
+  }
+  .row.destroying {
+    background-color: #ccc;
+    opacity: 0.2;
+    text-decoration: line-through;
+  }
```

Comment:
Yeah, not super important, the `git diff` tool just usually warns about it & github shows that red cross symbol at the bottom of its diff